docs/en/faq/osgi.md

```diff
@@ -0,0 +1,12 @@
+## How to make SkyWalking agent works in `OSGI` environment?
+
+`OSGI` implements its own set of [modularity](https://www.osgi.org/resources/modularity/), which means that each `Bundle` has its own unique class loader for isolating different versions of classes.
+By default, OSGI runtime uses the boot classloader for the bundle codes, which makes the `java.lang.ClassNotFoundException` exception in the booting stage.
```

Comment:
I reworded yours, could you add the typical error stack here? I want to be sure which ClassNotFound it is.